library/pkwrite.h

```diff
@@ -85,7 +85,7 @@
  *                                            + 2 * ECP_MAX (coords)    [1]
  *  }
  */
-#define MBEDTLS_PK_ECP_PUB_DER_MAX_BYTES    (30 + 2 * MBEDTLS_ECP_MAX_BYTES)
+#define MBEDTLS_PK_ECP_PUB_DER_MAX_BYTES    (30 + 2 * 66)
```

Comment:
That's the kind of thing I do in exploratory prototypes just to get moving, but it's not suitable for production code :)

We probably want to have a new macro `MBEDTLS_PK_ECC_MAX_BYTES` or something that would expand to either `MBEDTLS_ECP_MAX_BYTES` or its PSA equivalent (or max of both?) depending on the config.

Reply:
Ops, sorry, I looked at the changes in your drafted PR and it seemed OK to me, but I didn't correctly read the "direction" of the change (i.e. moving from a macro to a raw number). I thought it was the opposite ':)
BTW, I'll fix this of course

Reply:
Ah, I'm flattered, you assumed I wouldn't do something that dirty and then subconsciously reversed the direction of the change to compensate - sorry to disappoint ;) (just kidding of course)